kafka/admin/src/main/java/AdminConfiguration.java

```diff
@@ -93,9 +93,9 @@ public static Properties createProperties(AdminConfiguration config) {
             }
         }
 
-        if ((config.getOauthAccessToken() != null)
-                || (config.getOauthTokenEndpointUri() != null && config.getOauthClientId() != null && config.getOauthRefreshToken() != null)
-                || (config.getOauthTokenEndpointUri() != null && config.getOauthClientId() != null && config.getOauthClientSecret() != null))    {
+        if (config.getOauthAccessToken() != null
+                || config.getOauthTokenEndpointUri() != null && config.getOauthClientId() != null && config.getOauthRefreshToken() != null
+                || config.getOauthTokenEndpointUri() != null && config.getOauthClientId() != null && config.getOauthClientSecret() != null)    {
```

Comment:
Can we keep the old code? I think it makes the code more readable.

Reply:
Agree with Jakub

Reply:
This was forced by checkstyle plugin.
We can either leave it as it was and ignore the checkstyle error, 
or apply the checkstyle rules, your call.

Reply:
There was no checkstyle error before => so why you have checkstyle error now?

Reply:
Braces are back + added suppress ...